src/lib/libast/features/lib

```diff
@@ -7,6 +7,7 @@ hdr	locale,ndir,nl_types,process,spawn,syslog,utime,vfork
 hdr	linux/fs,linux/msdos_fs,sys/ioctl
 hdr	wchar note{ <wchar.h> and isw*() really work }end execute{
 	#include <wchar.h>
+	extern int iswalnum(wint_t);  /* work around implicit declaration warning or error */
```

Comment:
POSIX says this is defined by `<wctype.h>`. Does including that not work?

Reply:
I was using externs for compatibility reasons, although after some further testing it seems includes work fine.